.github/workflows/auto-merge-dependabot.yml

```diff
@@ -0,0 +1,32 @@
+name: Auto-merge dependabot updates
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
Our dependabot PRs target our default branch `dev`. Will push the changes across the PHP libs

Reply:
Thanks for the follow up @Ndiritu